src/components/create/PegInForm.vue

```diff
@@ -27,11 +27,17 @@
         </v-col>
       </v-row>
       <v-row class="mx-0">
-        <v-col cols="2" class="d-flex justify-start ma-0 pa-0">
+        <!-- <v-col cols="2" class="d-flex justify-start ma-0 pa-0">
```

Comment:
let's not keep this commented out. Remove it please

unless there is a reason for that, if so, leave the reason commented in the code as well